src/dfx/src/util/assets.rs

```diff
@@ -107,3 +107,205 @@ pub fn bitcoin_wasm(logger: &slog::Logger) -> DfxResult<Vec<u8>> {
         bail!("Failed to find bitcoin canister archive entry");
     }
 }
+
```

Comment:
Would you please add a comment here containing a URL (somewhere in https://github.com/dfinity/ic) to where this came from?

Reply:
Would this be okay? https://github.com/dfinity/interface-spec/blob/master/spec/_attachments/ic.did

Reply:
Follow up: it seems that the `did` file is written manually (and verified by the CI) so I believe `interface-spec` would be the closest thing to a source of truth there is.

Reply:
Sounds good.  Basically trying to find out: where should a maintainer look in order to find updates to this?

Reply:
The interface spec would be the place to get it from. I just forwarded you a thread on Slack